layouts/partials/commons/image.html

```diff
@@ -24,8 +24,16 @@
     {{- if .crop -}}
       {{- $crop = true -}}
     {{- end -}}
+    {{ $image_class := "" }}
+    {{ $is_png := strings.HasSuffix $image.name "png" }}
     {{ $is_svg := strings.HasSuffix $image.name "svg" }}
-    <picture {{ if $is_svg }}class="is-svg"{{ end }}>
+    {{ if $is_png }}
+      {{ $image_class = "is-png" }}
+    {{ else if $is_svg }}
+      {{ $image_class = "is-svg" }}
+    {{ end }}
+
+    <picture {{ if or $is_png $is_svg }}class="{{ $image_class }}"{{ end }}>
 
       {{- if strings.HasSuffix $image.name "svg" -}}
```

Comment:
le test du format est dupliqué ici

Reply:
Ah oui j'avais pas saisi l'endroit auquel tu faisais référence ! 
Est-ce que c'est mieux de faire `{{- if eq $image_class "is-svg" -}}` la 2e fois ?